FindPoetry.cmake

```diff
@@ -0,0 +1,35 @@
+#[=======================================================================[.rst:
+FindClang.cmake
```

Comment:
Should be updated

Reply:
https://github.com/Opentrons/cmake-utils/pull/24/commits/e43541b5a4b4af141dd8c058979647aef35c7357